packages/pwa-kit-dev/src/configs/webpack/config.js

```diff
@@ -212,6 +203,63 @@ const withChunking = (config) => {
     }
 }
 
+const ruleForBabelLoader = (babelPlugins) => {
+    return {
+        test: /(\.js(x?)|\.ts(x?))$/,
+        exclude: /node_modules/,
+        use: [
+            {
+                loader: findInProjectThenSDK('babel-loader'),
+                options: {
+                    rootMode: 'upward',
+                    cacheDirectory: true,
+                    ...(babelPlugins ? {plugins: babelPlugins} : {})
+                }
+            }
+        ]
+    }
+}
+
+const enableReactRefresh = (config) => {
+    const rules = [
+        ruleForBabelLoader(
+            [mode === development && require.resolve('react-refresh/babel')].filter(Boolean)
+        ),
+        ...config.module.rules.slice(1)
+    ]
+
+    return {
+        ...config,
+        module: {
+            ...config.module,
+            rules
+        },
+        entry: {
+            ...config.entry,
+            main: [mode === development && 'webpack-hot-middleware/client', './app/main'].filter(
```

Comment:
```suggestion
            main: [mode === development && 'webpack-hot-middleware/client?path=/mobify/bundle/development', './app/main'].filter(
```

For the double path issue, this might be what you need